### ISSUE TITLE
Recourse Packs

### ISSUE THREAD

**Issue by Tropfer** [Wurst-Imperium/Wurst-Client#1287]
If I go to options and press on Recourse Packs, minecraft crash 

Crash Report:

```
---------------- Wurst Client Crash Report ----------------
---- Please report this crash at: wurst-client.tk/bugs ----
// I blame Dinnerbone.

Time: 18.07.15 09:57
Description: Updating screen events

java.lang.IllegalArgumentException: input == null!
	at javax.imageio.ImageIO.read(Unknown Source)
	at net.minecraft.client.renderer.texture.TextureUtil.func_177053_a(TextureUtil.java:319)
	at net.minecraft.client.resources.DefaultResourcePack.getPackImage(DefaultResourcePack.java:95)
	at net.minecraft.client.resources.ResourcePackListEntryDefault.<init>(ResourcePackListEntryDefault.java:29)
	at net.minecraft.client.gui.GuiScreenResourcePacks.initGui(GuiScreenResourcePacks.java:71)
	at net.minecraft.client.gui.GuiScreen.setWorldAndResolution(GuiScreen.java:543)
	at net.minecraft.client.Minecraft.displayGuiScreen(Minecraft.java:1168)
	at net.minecraft.client.gui.GuiOptions.actionPerformed(GuiOptions.java:204)
	at net.minecraft.client.gui.GuiScreen.mouseClicked(GuiScreen.java:505)
	at net.minecraft.client.gui.GuiScreen.handleMouseInput(GuiScreen.java:591)
	at net.minecraft.client.gui.GuiScreen.handleInput(GuiScreen.java:560)
	at net.minecraft.client.Minecraft.runTick(Minecraft.java:1943)
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1272)
	at net.minecraft.client.Minecraft.run(Minecraft.java:383)
	at net.minecraft.client.main.Main.main(Main.java:127)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at javax.imageio.ImageIO.read(Unknown Source)
	at net.minecraft.client.renderer.texture.TextureUtil.func_177053_a(TextureUtil.java:319)
	at net.minecraft.client.resources.DefaultResourcePack.getPackImage(DefaultResourcePack.java:95)
	at net.minecraft.client.resources.ResourcePackListEntryDefault.<init>(ResourcePackListEntryDefault.java:29)
	at net.minecraft.client.gui.GuiScreenResourcePacks.initGui(GuiScreenResourcePacks.java:71)
	at net.minecraft.client.gui.GuiScreen.setWorldAndResolution(GuiScreen.java:543)
	at net.minecraft.client.Minecraft.displayGuiScreen(Minecraft.java:1168)
	at net.minecraft.client.gui.GuiOptions.actionPerformed(GuiOptions.java:204)
	at net.minecraft.client.gui.GuiScreen.mouseClicked(GuiScreen.java:505)
	at net.minecraft.client.gui.GuiScreen.handleMouseInput(GuiScreen.java:591)
	at net.minecraft.client.gui.GuiScreen.handleInput(GuiScreen.java:560)

-- Affected screen --
Details:
	Screen name: net.minecraft.client.gui.GuiScreenResourcePacks
Stacktrace:
	at net.minecraft.client.Minecraft.runTick(Minecraft.java:1943)
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1272)
	at net.minecraft.client.Minecraft.run(Minecraft.java:383)
	at net.minecraft.client.main.Main.main(Main.java:127)

-- System Details --
Details:
	Minecraft Version: 1.8
	Wurst Client Version: 2.1 (latest: 2.1)
	Operating System: Windows 7 (amd64) version 6.1
	Java Version: 1.8.0_31, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 109407280 bytes (104 MB) / 315736064 bytes (301 MB) up to 1060372480 bytes (1011 MB)
	JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx1G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	Launched Version: Wurst
	LWJGL: 2.9.1
	OpenGL: Intel(R) HD Graphics 4000 GL version 3.3.0 - Build 8.15.10.2778, Intel
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: No
	Is Modded: Very likely; Jar signature invalidated
	Type: Client (map_client.txt)
	Resource Packs: [StegPack.zip]
	Current Language: English (US)
	Profiler Position: N/A (disabled)
```

**Comment by ayciate** [comment]
I can confirm this bug :+1:

**Comment by mcccs** [comment]
Yeah , i saw this crash Rubix hack

**Comment by trollmad3** [comment]
this just happend to me DX

**Comment by WurstContributor** [comment]
Hard to read xD?

**Comment by ValenciaOrange** [comment]
I got this error
---------------- Wurst Client Crash Report ----------------
---- Please report this crash at: wurst-client.tk/bugs ----
// Ooh. Shiny.

Time: 8/10/15 7:00 PM
Description: Updating screen events

java.lang.IllegalArgumentException: input == null!
	at javax.imageio.ImageIO.read(ImageIO.java:1348)
	at net.minecraft.client.renderer.texture.TextureUtil.func_177053_a(TextureUtil.java:319)
	at net.minecraft.client.resources.DefaultResourcePack.getPackImage(DefaultResourcePack.java:95)
	at net.minecraft.client.resources.ResourcePackListEntryDefault.<init>(ResourcePackListEntryDefault.java:29)
	at net.minecraft.client.gui.GuiScreenResourcePacks.initGui(GuiScreenResourcePacks.java:71)
	at net.minecraft.client.gui.GuiScreen.setWorldAndResolution(GuiScreen.java:543)
	at net.minecraft.client.Minecraft.displayGuiScreen(Minecraft.java:1168)
	at net.minecraft.client.gui.GuiOptions.actionPerformed(GuiOptions.java:204)
	at net.minecraft.client.gui.GuiScreen.mouseClicked(GuiScreen.java:505)
	at net.minecraft.client.gui.GuiScreen.handleMouseInput(GuiScreen.java:591)
	at net.minecraft.client.gui.GuiScreen.handleInput(GuiScreen.java:560)
	at net.minecraft.client.Minecraft.runTick(Minecraft.java:1943)
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1272)
	at net.minecraft.client.Minecraft.run(Minecraft.java:383)
	at net.minecraft.client.main.Main.main(Main.java:127)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at javax.imageio.ImageIO.read(ImageIO.java:1348)
	at net.minecraft.client.renderer.texture.TextureUtil.func_177053_a(TextureUtil.java:319)
	at net.minecraft.client.resources.DefaultResourcePack.getPackImage(DefaultResourcePack.java:95)
	at net.minecraft.client.resources.ResourcePackListEntryDefault.<init>(ResourcePackListEntryDefault.java:29)
	at net.minecraft.client.gui.GuiScreenResourcePacks.initGui(GuiScreenResourcePacks.java:71)
	at net.minecraft.client.gui.GuiScreen.setWorldAndResolution(GuiScreen.java:543)
	at net.minecraft.client.Minecraft.displayGuiScreen(Minecraft.java:1168)
	at net.minecraft.client.gui.GuiOptions.actionPerformed(GuiOptions.java:204)
	at net.minecraft.client.gui.GuiScreen.mouseClicked(GuiScreen.java:505)
	at net.minecraft.client.gui.GuiScreen.handleMouseInput(GuiScreen.java:591)
	at net.minecraft.client.gui.GuiScreen.handleInput(GuiScreen.java:560)

-- Affected screen --
Details:
	Screen name: net.minecraft.client.gui.GuiScreenResourcePacks
Stacktrace:
	at net.minecraft.client.Minecraft.runTick(Minecraft.java:1943)
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1272)
	at net.minecraft.client.Minecraft.run(Minecraft.java:383)
	at net.minecraft.client.main.Main.main(Main.java:127)

-- System Details --
Details:
	Minecraft Version: 1.8
	Wurst Client Version: 2.1 (latest: 2.1)
	Operating System: Windows 7 (amd64) version 6.1
	Java Version: 1.8.0_25, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 19638904 bytes (18 MB) / 247697408 bytes (236 MB) up to 1060372480 bytes (1011 MB)
	JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx1G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	Launched Version: Wurst
	LWJGL: 2.9.1
	OpenGL: Intel(R) HD Graphics GL version 2.1.0 - Build 8.15.10.2119, Intel
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because EXT_framebuffer_object is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: Yes
	Is Modded: Very likely; Jar signature invalidated
	Type: Client (map_client.txt)
	Resource Packs: [OmegaPack2.zip, PACK.zip]
	Current Language: English (US)
	Profiler Position: N/A (disabled)

**Comment by Alexander01998** [comment]
Fixed. It was the compiler not putting `pack.png` into the Jar.
